app/views/shared/_secondary_navigation.html.erb

```diff
@@ -54,21 +54,24 @@
             </li>
           <% end %>
         <% elsif project_page? %>
-          <% if can_read_project?(@project) then %>
-            <li id="canvas-nav-tab" class="<%= "active" if is_project_show? %>">
-              <a href="<%= project_url(@project) %>" title="<%=t "nav2.projects.show" %>">
-                <span class="hidden-sm hidden-md"><%=t "nav2.projects.show" %></span>
-                <span class="hidden-xs hidden-lg fas fa-folder"></span>
-              </a>
-            </li>
-            <li id="project-archive-nav-tab" class="<%= "active" if is_project_archive? %>">
-              <a href="<%= experiment_archive_project_url(@project) %>" title="<%=t "nav2.projects.archive" %>">
-                <span class="hidden-sm hidden-md"><%=t "nav2.projects.archive" %></span>
-                <span class="hidden-xs hidden-lg fas fa-briefcase"></span>
-              </a>
-            </li>
-          <% end %>
-
+          <div class="dropdown pull-left">
+            <button class="btn btn-light icon-btn dropdown-toggle" type="button" id="sortMenu" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+              <i class="fas fa-sort-amount-up"></i>
+            </button>
+            <ul class="dropdown-menu" aria-labelledby="sortMenu">
+              <% ["new", "old", "atoz", "ztoa", "arch_new", "arch_old"].each do |sort| %>
+                <% unless action_name != 'experiment_archive' && sort.include?('arch') %>
+                  <li>
+                    <% if @current_sort != sort %>
+                      <a href="?<%= {sort: sort}.reject{|v| v.to_s == "0"}.to_query %>"><%= t("general.sort.#{sort}_html") %></a>
+                    <% else %>
+                      <a class="disabled" href="#"><%= t("general.sort.#{sort}_html") %></a>
```

Comment:
Same here, not sure how sorting looks like in zeplin for project, but  it was changed for project index (label with checkmark for selected item)